llvm/tools/verify-uselistorder/verify-uselistorder.cpp

```diff
@@ -545,18 +540,11 @@ int main(int argc, char **argv) {
   cl::ParseCommandLineOptions(argc, argv,
                               "llvm tool to verify use-list order\n");
 
-  // Do not load bitcode into the new debug info format by default.
-  if (LoadBitcodeIntoNewDbgInfoFormat == cl::boolOrDefault::BOU_UNSET)
-    LoadBitcodeIntoNewDbgInfoFormat = cl::boolOrDefault::BOU_FALSE;
-
   LLVMContext Context;
   SMDiagnostic Err;
 
   // Load the input module...
   std::unique_ptr<Module> M = parseIRFile(InputFilename, Err, Context);
-  // verify-uselistoder currently only supports old-style debug info mode.
-  // FIXME: Update mapping code for RemoveDIs.
-  M->setIsNewDbgInfoFormat(false);
```

Comment:
These used to be asserts (checking we're not using the new mode). IMO we should consider adding those back in again until verify-uselistorder actually supports the new format, as AFAIUI right now the DbgRecords are just ignored (not mapped, i.e., their ordering isn't checked by verify-uselistorder). WDYT?